.github/workflows/benchmarks-run.yml

```diff
@@ -80,17 +65,5 @@ jobs:
         id: commit-updated-weights
         uses: stefanzweifel/git-auto-commit-action@8756aa072ef5b4a080af5dc8fef36c5d586e521d
         with:
-          commit_message: "Update weights for #${{github.event.inputs.branch}}"
+          commit_message: "Update weights"
```

Comment:
nit: why remove the `github.event.inputs.branch`?

Reply:
A few reasons:
1. Might eventually want to run this on any ref
2. We squash, so it doesn't have long term value
3. If you switch branch names it can get confusing.